src/test/routes/features/agreement/controllers/lot.test.ts

```diff
@@ -0,0 +1,85 @@
+//@ts-nocheck 
+
+import { expect } from 'chai';
+import request from 'supertest';
+import { app } from '../../../../../main/app';
+import nock from 'nock'
+import express from 'express';
+
+describe('Choose a lot agreement page', () => {
+  let parentApp;
+  const dummyJWT = 'eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJpYXQiOjE2Mzc3NTU1NTUsImp0aSI6IjQ0YWVjN2Q0LTAxY2UtNGEwNS05MWE3LTU3ZWExNmUxZjFjNiIsInVpZCI6IjEwMzgiLCJjYWxsZXIiOiJ1c2VyIiwiY2lpT3JnSWQiOiI1NTY4OTE2MzM2MTk4NTE2MDMiLCJyb2xlcyI6WyJDQVRfQURNSU5JU1RSQVRPUiIsIkNBVF9VU0VSIl0sInN1YiI6ImNlc2FyLmZlcm5hbmRlekBjb2duaXphbnQuY29tIiwiZXhwIjoxNjM3NzU2NDU1LCJpc3MiOiJDQ1MiLCJhdWQiOiJ6eWljckRhMG9Kc0g0aFVMSVdUTmRhZHhRVjQ3N3c0NSJ9.eveTQA_33H2WOWig1eK0O1n8kOODQCUrXVQWxGaI-zVKw4Jjh9FPX9dzireNGzLxfkJYlb1IOW-4LR2g6LldUIs2RY9-2-rBo9ZIGTgcRjgR_uvM-UnhMTfbuTJWfDK8r7XQ1A4qNOywlAQ8xDjhNzrHwA8ZreFJVE6ijiYJHdMyNgIwJKtvNLE9k1TksPf0eDw90QhYMI3uJaAg4wRDgNVV8gyr3Gacsac6phCeKlfMJRlzrS_k4OcKG9nqQgfpHhqvvO-58TUY72rPAWJQ7m4xSiFXD2B8VR-n7ZgleRyvQb3K47Eb-4Tiu70kcPxtEcczxhQTfZB8BpUg3-y_hA'
```

Comment:
This consit of personal data which is being exposed

Reply:
removed